composer.json

```diff
@@ -16,9 +16,9 @@
     },
     "require-dev": {
         "composer/composer": "^2.1",
+        "phpunit/phpunit": "^12.4",
         "symfony/dotenv": "^6.4|^7.4|^8.0",
         "symfony/filesystem": "^6.4|^7.4|^8.0",
-        "symfony/phpunit-bridge": "^6.4|^7.4|^8.0",
```

Comment:
the SymfonyExtension defined in the bridge still does the automatic registration of the DebugClassLoader to report deprecations triggered by this autoloader. If the bridge is not used anymore, it might still make sense to depend on error-handler and to register that autoloader in the PHPUnit bootstrap file.